database/migrations/2016_05_02_173914_move_featured_reportback_to_competition.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class MoveFeaturedReportbackToCompetition extends Migration
```

Comment:
Deleting migrations after they've run before will break your migrations. _I may know this from experience..._